.pre-commit-config.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+default_stages: [commit, push]
+minimum_pre_commit_version: "1.20.0"
+repos:
+  - repo: https://github.com/Lucas-C/pre-commit-hooks
```

Comment:
This doesn't resolve -- what should it be?

Reply:
It works for me. https://github.com/Lucas-C/pre-commit-hooks
Similar code exists in other Google projects: 
https://github.com/google/triage-party/blob/master/.pre-commit-config.yaml
https://github.com/GoogleCloudPlatform/oozie-to-airflow/blob/master/.pre-commit-config.yaml

Reply:
Weird, I could swear I got a 404 yesterday. Sorry then!